exercicioDeHoje.js

```diff
@@ -0,0 +1,5 @@
+function Cacique(rappi) {
+  if (rappi === 'hamburguer') {
+    console.log('Xablau');
+  }
+}
```

Comment:
Parabens, mas eu acho que podemos colocar mais um print
```suggestion
function Cacique(rappi) {
  if (rappi === 'hamburguer') {
    console.log('Xablau');
    console.log('E agora eu quero um KitKat');
  }
}
```